frontend/src/components/grouping/FuzzyMatchNotificationGroupingComponent.vue

```diff
@@ -104,11 +106,27 @@ const label = (matchedFieldsText: string, warning: boolean) => {
   return `${prefix} ${matchedFieldsText}`;
 }
 
+const getErrorsItemContent = ref((matches: MiscFuzzyMatchResult[]) => {
+
+  const matchValue = matches && matches.length > 0 ? getUniqueClientNumbers(matches) : [];    
+  const nonFuzzyMatch = matches.find((match) => !match.result?.fuzzy);
+
+  return nonFuzzyMatch && nonFuzzyMatch.result?.match ? renderErrorItem(
+    {
+      ...nonFuzzyMatch,
+      result: { ...nonFuzzyMatch.result, match: matchValue.join(",") },
+    }) : "";
+});
+
+const getErrorItemContent = ref((match: MiscFuzzyMatchResult) => {
+  return match && match.result?.match ? renderErrorItem(match) : "";
+});
```

Comment:
Maybe we should remove this function since it's never used.

Reply:
Good catch, my VSCode wasn't pointing out this